<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataRecord</class>
 <widget class="QWidget" name="DataRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:disabled
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:pressed
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 0, 255); 
	border:2px solid white; 
	border-left-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, 
                        stop: 0 gray, stop: 1 black); 			border-top-color: qlineargradient(x1: 0, y1: 0, x2: 0, 										y2: 1, stop: 0 gray, stop: 1 black);
  border-right-color: white; border-bottom-color: white;
}</string>
  </property>
  <widget class="QPushButton" name="btnRecordSet">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>90</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>数据
设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>450</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>82</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>累计参数名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>130</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>里程</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>178</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>1单元压缩机工作时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>226</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>2单元压缩机工作时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>274</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>总能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>322</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>辅助能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>370</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>牵引能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>418</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>电阻制动能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>466</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>再生电量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>82</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>总累计量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrakeComsuption1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>418</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegeneratePower1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>466</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSivComsuption1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>322</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTcuComsuption1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>370</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompressor1Time1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>178</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompressor2Time1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>226</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAllConsumption1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>274</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblMileage1">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>130</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTime">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>82</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>最近运行30天累计量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAllConsumption2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>274</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSivComsuption2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>322</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrakeComsuption2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>418</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompressor1Time2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>178</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompressor2Time2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>226</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTcuComsuption2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>370</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegeneratePower2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>466</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblMileage2">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>130</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDistance">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>82</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>最近运行1000Km累计量</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblAllConsumption3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>274</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSivComsuption3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>322</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBrakeComsuption3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>418</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompressor1Time3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>178</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCompressor2Time3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>226</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTcuComsuption3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>370</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegeneratePower3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>466</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblMileage3">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>130</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>82</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 230);
color: black;</string>
   </property>
   <property name="text">
    <string>单位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_57">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>274</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Kw•h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>322</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Kw•h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>418</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Kw•h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_60">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>178</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_61">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>226</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_62">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>370</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Kw•h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_63">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>466</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Kw•h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_64">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Km</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_107">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>0</y>
     <width>2</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTip">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>821</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
color: red;</string>
   </property>
   <property name="text">
    <string>请在司机室占有条件下，进行数据设置操作！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
